SDPi_Supplement/asciidoc/sdpi-supplement.adoc

```diff
@@ -16,6 +16,8 @@
 :ihe_supplement_sdpi_revision_date: {localdatetime}
 :ihe_supplement_sdpi_revision_label: Draft in Preparation for Public Comment
 
+{empty} +

```

Comment:
If we want to produce space, we should prefer CSS style ober blank lines

Reply:
suggest the change and we'll use it instead of what I added - the logos were just jammed at the top of the window ... a little space would be good

Reply:
Added a #TODO comment to the AsciiDoc source.